### PR TITLE
Added --large modifier for label

### PR DESCRIPTION
**Pull request by matzimowski**
closes https://github.com/brainly/style-guide/issues/180

screen shot 2015-09-22 at 10 41 58

### CODE REVIEW

File: src/components/labels/_labels.scss [hunk 1]
```diff
@@ -136,6 +138,25 @@ $includeHtml: false !default;
         font-size: $labelIconSizeSecondary;
       }
     }
+
+    &--large {
+      .mint-label__text,
+      .mint-label__number {
+        @include fixText($labelFontSizeLarge);
+        text-transform: none;
+
```
Comment (vergilius) [inline src/components/labels/_labels.scss]: lots of whitespaces here, do we have an setting in style-checking for that?

**Comment by aju** [comment]
:ok:

File: src/components/labels/_labels.scss (hunk 1 above)
Comment (matzimowski) [inline src/components/labels/_labels.scss]: just one additional whitespace, i'll fix it...

Comment (aju) [inline src/components/labels/_labels.scss]: Typical Mati :point_right: :point_left:

**Comment by vergilius** [comment]
:+1:

**Pull request merged**